global-news-app/src/assets/styles/global-styles.ts

```diff
@@ -0,0 +1,66 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+
+html {
+    box-sizing: border-box;
+    overflow-x: hidden;
+    overflow-y: overlay;
```

Comment:
what does `overflow-y: overlay` do?

Reply:
Turns out it's deprecated, I'll delete it